lgc/patch/FragColorExport.cpp

```diff
@@ -46,40 +46,21 @@ namespace lgc {
 //
 // @param pipelineState : Pipeline state
 // @param module : LLVM module
-FragColorExport::FragColorExport(PipelineState *pipelineState, Module *module)
-    : m_pipelineState(pipelineState), m_context(module ? &module->getContext() : nullptr) {
+FragColorExport::FragColorExport(LLVMContext *context) : m_context(context) {
 }
 
 // =====================================================================================================================
 // Executes fragment color export operations based on the specified output type and its location.
 //
 // @param output : Fragment color output
-// @param location : Location of fragment color output
+// @param hwMrt : The render target (MRT) of fragment color output
```

Comment:
I prefer to use colorTarget. We have use MaxColorTargets in LGC to represent this idea.

Reply:
Steven and I had a separate discussion about this on a different PR due to confusion about location vs. originalLocation elsewhere in the code. I find the `hw` prefix useful to be absolutely clear on what this indexes (unlike some of the "location", due to compaction and also the potential for confusion with how dual source blend works). I was slightly leaning towards "MRT" because that's what the hardware uses, though the MaxColorTargets point you make is good. I don't feel particularly strongly about the MRT vs. ColorTarget thing, so `hwColorTarget` (or just `hwTarget`?) would be fine with me.

Reply:
I just feel `mrt` is too abstract for someone who is not familiar with this hardware nomeclature, so I suggest the change of it because color attachment is widely used in Vulkan spec. It is natural to link it with color target. It is OK to add the `hw `prefix.